src/dbally/views/methods_base.py

```diff
@@ -99,3 +109,19 @@ async def call_filter_method(self, func: syntax.FunctionCall) -> Any:
         if inspect.iscoroutinefunction(method):
             return await method(*args)
         return method(*args)
+
+    async def call_aggregation_method(self, func: syntax.FunctionCall) -> Any:
```

Comment:
Do we need a separate `call_aggregation_method`? It seems that we could just rename `call_filter_method` to `call_method` since it seems that both the body, the documentation, and the purpose of both is the same.

Reply:
I like the separate method here because we are also doing the decorator validation here, which is specific to the aggregation method. We could add that as an argument here, but it would make the interface a bit ambiguous, imho. But maybe I could create a private method inside the `MethodsBasedView` just for method calls, since this is a common part and use it in `call_filter_method` and `call_aggregation_method`.